### ISSUE TITLE
(AA) Non-text colour contrast requirements not met

### ISSUE THREAD

**Issue by otis-bath** [open-sdg/open-sdg#1760]
**Medium level (AA)**

https://sdgdata.gov.uk/
Across the site, on the homepage, goals page, indicator page etc. when an object is selected a yellow border appears around the outside. The yellow (#ffdd00) border used to indicate when the links are focused fails colour contrast guidelines with a ratio of 1.3:1 against the white background.
![image](https://user-images.githubusercontent.com/90769108/182163680-ede8d01f-98a5-4ed9-b118-3dd6edc3cc97.png)
![image](https://user-images.githubusercontent.com/90769108/182163784-2d6bd0c9-b10e-4685-91a2-c9b02b0aa489.png)
![image](https://user-images.githubusercontent.com/90769108/182163400-fe341b79-407e-48a5-92fa-bb0c9e4cefe6.png)

We need this ratio to be at least 3:1

**Comment by brockfanning** [comment]
@phillipgardiner Do you have any ideas on what darker color to use?

A bit of background also - in Open SDG we use a [focus outline color](https://github.com/open-sdg/open-sdg/blob/2.1.0-dev/_sass/variables/_colors.scss#L7) and a [focus background color](https://github.com/open-sdg/open-sdg/blob/2.1.0-dev/_sass/variables/_colors.scss#L6), which both happen to be the same: FFDD00. The focus outline color is used on things like goal icons, the chart, the main logo, and the search bar. And the focus background color is used on things like links, tabs, and menu items.

My interpretation of this issue is that we need to make the outline color darker. But maybe we can keep the background color at FFDD00? (Or maybe we should make them both darker?)

**Comment by phillipgardiner** [comment]
@brockfanning Sorry been away on holiday. Has this now been dealt with at all? I'm not 100% certain on this. The #FFDD00 yellow we use is what is stated by GDS in their standards so surprised it is not compliant. They have some guidelines here: https://design-system.service.gov.uk/get-started/focus-states/. I notice that on things like our search box we don't make the black outline thicker (like in the standard) might this improve things? Perhaps we can pick this up in the tech call tomorrow?

**Comment by otis-bath** [comment]
Brock to setup the black and yellow border for the selected option (search, graph selected area, goal tiles)
